compiler/rustc_typeck/src/check/method/probe.rs

```diff
@@ -170,6 +170,10 @@ pub struct Pick<'tcx> {
     ///     B = A | &A | &mut A
     pub autoref: Option<hir::Mutability>,
 
+    /// Only valid when the self type is a `*mut`: coerce the receiver to `*const`. If this it
+    /// `true` then `autoref` should be `None`.
```

Comment:
Nice comment :)

Reply:
I reworded this slightly, saying "Only valid when the receiver type ..." instead of "... self type ...".